src/cryptography/hazmat/primitives/serialization/pkcs12.py

```diff
@@ -4,6 +4,17 @@
 
 from __future__ import absolute_import, division, print_function
 
+from cryptography.hazmat.backends import default_backend
+
 
 def load_key_and_certificates(data, password, backend):
     return backend.load_key_and_certificates_from_pkcs12(data, password)
+
+
+def serialize_key_and_certificates(name, key, cert, cas, key_encryption):
```

Comment:
You removed cert_encryption. See above as to why both should exist (please discuss this with Paul)

Reply:
I think we're going to see what happens with a relatively aggressive default and then we can expand support from there. One of the challenges in this space is that everyone leaves everything in terrible legacy modes even when it's not necessary because they're afraid of breakage. In this case we can be (relatively) aggressive at first and then expand compatibility (if necessary) without breaking existing consumers.